src/applications/vaos/actions/confirmed.json

```diff
@@ -0,0 +1,28 @@
+[{
```

Comment:
Did you pull this from staging?

Reply:
This was just based on the schema I created for the API contract.  I haven't been able to log into staging the past couple times I tried to mock the current response. I am thinking we can adjust this mock data as necessary once we get to the details page or betamocks is in place.

Reply:
Works for me